src/auth_manifests.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: training-operator
+  name: {{ namespace }}-training-operator
```

Comment:
The `training-operator` part could also change depending on the application name with which it gets deployed. I would recommend templating the application name also. Something like:
```suggestion
    app: {{ app-name }}
  name: {{ namespace }}-{{ app-name }}
```

Reply:
That's true, good catch! I'll change it (:

Reply:
Updated in latest version. Resolving this.